index.html

```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
+<html lang="en-US">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device0width">
+  <title></title>
+</head>
+
+<body>
+  <header>
+    <nav>
+      <ul>
+        <li><a>the offer</a></li>
+        <li><a>portfolio</a></li>
+        <li><a>about us</a></li>
+        <li><a>contact</a></li>
+      </ul>
+    </nav>
+    <section>
+      <h2>hello!</h2>
+      <p>
+        We are creative agency based in Warsaw, Poland <br>
+        Creating projects for clients all over the world
+      </p>
+    </section>
+    <section>
+      <h2>design!</h2>
+      <p>
+        Good design is our domain <br>
+        We work in multidisciplinary teams.
+      </p>
+    </section>
+    <section>
+      <h2>projects!</h2>
+      <p>
+        We always select varied projects, <br>
+        to step out of comfort zone.
+      </p>
+    </section>
+  </header>
+
+  <main>
+    <p>scroll down to find out more about our studio</p>
+    <article>
```

Comment:
I'd say it's a theme, so I'd go with `<section>` for works and then `<article>` for a single piece of work and then I think `<div>` for containers within - this can still change due to styling needs.